starwars/src/App.js

```diff
@@ -24,12 +27,17 @@ class App extends Component {
       });
   }
   render() {
+    this.stateItems = this.state.starwarsChars.map((item, i) =>
+      <div key={i}>{item.name}</div>
+    );
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <div>{this.stateItems}</div>
```

Comment:
It'd be nice to see if there is a bit more you could render out than just the name of the star wars char. This is where it may be nice to build your own component to handle these values and properties